openshift/templates/base-images/postgres-14.yaml

```diff
@@ -48,9 +44,9 @@ objects:
         source: dockerhub
     spec:
       tags:
-        - name: "14"
+        - name: "14.4-bullseye"
```

Comment:
Just curious, what's the benefit of using Debian bullseye with postgres?

Reply:
This was what was here, just updating the version